ccd-definition/CaseTypeTab.json

```diff
@@ -14,18 +14,28 @@
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "Channel": "CaseWorker",
     "TabID": "HearingTab",
-    "TabLabel": "Hearing",
-    "TabDisplayOrder": 2,
+    "TabLabel": "Hearings",
```

Comment:
Changed as part of this PR as noticed on Eilis prototype and it was supposed to be Hearings. Check with Adam and ok for this PR.

Reply:
@Becky please get this added to AC so QA can test

Reply:
No problem going to speak to Moray about all the changes now :)

Reply:
I see ACs are updated.